src/screens/Home/components/PopularSingleCard/PopularSingleCard.tsx

```diff
@@ -6,28 +6,39 @@ import {
   TouchableOpacity
 } from 'react-native'
 import { Heading, HeadingType, PlainText } from 'src/components'
+import { EVEN_RATIO } from './common/constants'
 import styles from './styles'
 
 type Props = {
+  id: number
+  position: number
   title: string
   author: string
   source?: string
   style?: ImageStyle
-  onPress?: () => void
+  onPress: (author: string, id: number) => void
 }
 
 const PopularSingleCard: React.FC<Props> = ({
+  id,
+  position,
   title,
   author,
   source,
   style,
   onPress
 }) => {
+  const isCardRight = Boolean(position % EVEN_RATIO !== 0)
```

Comment:
Let's avoid this calculation and do it by styles. I think `flexWrap` should help you.